weave/trace_server/sqlite_trace_server.py

```diff
@@ -991,6 +989,8 @@ def feedback_create(self, req: tsi.FeedbackCreateReq) -> tsi.FeedbackCreateRes:
             req.payload["detoned"] = detoned
             req.payload["detoned_alias"] = emoji.demojize(detoned)
             res_payload = req.payload
+        else:
```

Comment:
nit: put res_payload = req.payload outside of the conditional if both branches do it